### PR TITLE
updated makefile to handle development zip

### PR DESCRIPTION
**Pull request by AlexChaseJones**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,3 +32,19 @@ dist: clean build pack
 
 dist-custom: clean build custom-pack
 
+dev-packer:
```
Comment (dagifted1) [inline Makefile]: should be pack instead of packer to be consistent

File: Makefile [hunk 2]
```diff
@@ -32,3 +32,19 @@ dist: clean build pack
...
+	mkdir -p ./dist
+	zip -r app_`date -u +"%Y-%m-%dT%H:%M:%SZ"`.zip * . && \
```
Comment (dagifted1) [inline Makefile]: should be hyphens instead of snake i.e. "app-" instead of "app_"

File: Makefile [hunk 3]
```diff
@@ -32,3 +32,19 @@ dist: clean build pack
...
+	cp app_*.zip ./dist && \
```
Comment (dagifted1) [inline Makefile]: see previous comment

File: Makefile [hunk 4]
```diff
@@ -32,3 +32,19 @@ dist: clean build pack
...
+	rm app_*.zip
```
Comment (dagifted1) [inline Makefile]: see previous comment

File: Makefile [hunk 5]
```diff
@@ -32,3 +32,19 @@ dist: clean build pack
...
+
+dev-packer-custom:
+	mkdir -p ./dist
+	zip -r $(app)_`date -u +"%Y-%m-%dT%H:%M:%SZ"`.zip * . && \
+	cp $(app)_*.zip ./dist && \
+	rm $(app)_*.zip
+
+dev-dist: clean lint dev-packer
```
Comment (dagifted1) [inline Makefile]: change packer to pack

File: Makefile [hunk 6]
```diff
@@ -32,3 +32,19 @@ dist: clean build pack
...
+
+dev-dist-custom: clean lint dev-packer-custom
```
Comment (dagifted1) [inline Makefile]: change packer to pack

File: Makefile [hunk 7]
```diff
@@ -32,3 +32,19 @@ dist: clean build pack
 
 dist-custom: clean build custom-pack
 
+dev-packer:
+	mkdir -p ./dist
+	zip -r app_`date -u +"%Y-%m-%dT%H:%M:%SZ"`.zip * . && \
+	cp app_*.zip ./dist && \
+	rm app_*.zip
+
+dev-packer-custom:
```
Comment (dagifted1) [inline Makefile]: copy changes from dev-pack

**Comment by dagifted1** [comment]
Also need to zip hidden files for .eslintrc.json and .gitignore.
Exclude node_modules, .DS_Store and dist folder.
Remove and delete dist folder and its contents

**Comment by maborowsky** [comment]
Verified that 'make dev-dist' returns the exact same result as zipping a new clone of this repo

**Pull request merged**